scripts/docs.py

```diff
@@ -439,5 +430,51 @@ def get_sections(nav: list) -> Dict[Tuple[str, ...], str]:
     return sections
 
 
+def get_sections_all(nav: list) -> Dict[Tuple[str, ...], Set[str]]:
+    sections = {}
+    for item in nav:
+        if type(item) is str:
+            continue
+        elif type(item) is dict:
```

Comment:
You added the continue keyword to the above if condition. Make it if and [reverse the condition](https://szymonkrajewski.pl/why-should-you-return-early/) and add continue. You can also merge two conditions like the following.
```py
if type(item) is str or type(item) is not dict:
    continue
```